tests/integration/errata_tool_product_version/create-2.yml

```diff
@@ -0,0 +1,47 @@
+- name: Add Create2 Product Version
+  errata_tool_product_version:
+    product: RHEL
+    name: RHEL-CREATE-2
+    description: Red Hat Create2 Product Version
+    default_brew_tag: rhel-8.0.0-candidate
+    rhel_release_name: RHEL-8.0.0
+    push_targets: []
+    is_rhel_addon: false
+    brew_tags: [rhel-8.0.0-candidate]
+    allow_rhn_debuginfo: true
+    is_oval_product: true
+    allow_buildroot_push: true
+    is_server_only: false
+    use_quay_for_containers: false
+    use_quay_for_containers_stage: false
+
+# We cannot get the name directly yet, CLOUDWF-3
+- name: query API for this product version
+  errata_tool_request:
+    path: api/v1/products/RHEL/product_versions/?filter[name]=RHEL-CREATE-1
+  register: response
+
+- name: parse product version JSON
+  set_fact:
+    attributes: "{{ response.json.data.0.attributes }}"
+    relationships: "{{ response.json.data.0.relationships }}"
+
+- assert:
+    that:
+      - attributes.name == 'RHEL-CREATE-1'
```

Comment:
```suggestion
      - attributes.name == 'RHEL-CREATE-1'
```

Reply:
Hi I am looking into it and I think you made a accidental mistake and there should be `- attributes.name == 'RHEL-CREATE-2'`. Am I right?